Sources/DateScrubber.swift

```diff
@@ -69,20 +81,48 @@ public class DateScrubber: UIViewController {
         }
     }
 
+    private var sectionLabelState = State.Inactive {
+        didSet {
+
+            if self.sectionLabelState == .Active {self.setSectionLabelActive()}
+            if self.sectionLabelState == .Inactive {self.setSectionLabelInactive()}
+            self.updateSectionLabelFrame()
+        }
+    }
+
+    private var scrubberState = State.Inactive {
+        didSet {
+            self.updateDateScrubberFrame()
+        }
+    }
+
     override public func viewDidLoad() {
         super.viewDidLoad()
 
         self.setSectionlabelFrame()
         self.view.addSubview(self.sectionLabel)
 
         self.dragGestureRecognizer.addTarget(self, action: #selector(handleDrag))
+        self.dragGestureRecognizer
         self.scrubberImageView.userInteractionEnabled  = true
         self.scrubberImageView.addGestureRecognizer(self.dragGestureRecognizer)
+
+        self.longPressGestureRecognizer.addTarget(self, action: #selector(handleLongPress))
+        self.longPressGestureRecognizer.minimumPressDuration = 0.2
+        self.longPressGestureRecognizer.cancelsTouchesInView = false
+        self.longPressGestureRecognizer.delegate = self
+
+
+        let scrubberGestureView = UIView(frame: CGRectMake(self.containingViewFrame.width-44,0,44,self.viewHeight))
```

Comment:
This makes two out of three developers so far. Maybe I should give something like a quick talk on auto-layout? 😅